ACT-reqs.md

```diff
@@ -0,0 +1,84 @@
+ACT Framework Requirements
+==========================
+
+Abstract
+--------
+
+The Accessibility Conformance Testing Framework will be a W3C specification, developed by the ACT Taskforce, lead by the WCAG Working Group. This document outlines the requirements for the ACT Framework. The ACT Framework is one part of the deliverables of the ACT Taskforce, for more details on this read [[ACT Deliverables]].
+
+Status of This Document
+-----------------------
+
+This is the first draft of this document, it has not yet had the full review of the ACT Taskforce. It is therefore likely to change significantly.
+
+
+1. Introduction
+---------------
+
+There are many accessibility test tools (ATTs). These can be fully automatic, or can prompt a user for specific actions, in order to discover accessibility issues of some digital product. Each of these tools will run one, or a serie of tests to come to some result. These test procedures may be designed to look for violations of WCAG 2.0, or some other accessibility standard, and for some that are highly customizable they can even be used in testing an organization's own accessibility policy. The test procedures used by ATT are often called 'rules'.
+
+The ACT Framework will standardize those aspects all rules have in common that ensure them to be reliable, implementable by different ATTs and transparent for the users of this tool. By standardizing this, the ACT Taskforce aims to promote a common understanding of how accessibility can be tested for certain technologies.
+
+Simply put, the ACT Framework is a standard on how to write test rules. Rules written in conformance to the ACT Framework specification are referred to as ACT Rules. These in turn are test procedures for accessibility, that can be implemented by ATTs or used for testing of accessibility.
+
+2. Requirements
+---------------
+
+### 2.1 Ensure Readability
+
+The ACT Framework should ensure that the rules are readable by both tool developers and less-technical users. For instance, the ACT Framework may discourage the use of pseudo code in favor of plain English.
+
+### 2.2 Ensure Justification
+
+The ACT Framework should ensure that ACT Rules are well grounded in the requirements they test. It is insufficient for an ACT Rule to merely claim something violates a requirement. The ACT Framework has to ensure that ACT Rules document why its claims are a valid interpretation of the requirement.
+
+### 2.3 Ensure Consistency
+
+The ACT Framework has to ensure that ACT Rules can be implemented in a way that provides consistent results. An ACT Rule must include a way to test the implementation of that rule.
+
+### 2.4 Ensure Measurable Accuracy
+
```

Comment:
*measurable* sounds confusing to me (maybe it's just me, as a non-native speaker), as it can have both a strict meaning (evaluate a quantity/size/amount, with units) or a laxer one (assess the importance/value).
I'm not sure the prose below clarifies that (to be discussed).